src/lib/span.ts

```diff
@@ -0,0 +1,38 @@
+export class Span {
```

Comment:
Classes that share the name of the file should use a default export.

```ts
class Span {}

export default Span;

```

All exports are at the bottom of the file.